lib/SearchAndSort/SearchAndSort.js

```diff
@@ -45,6 +44,7 @@ import {
   AppIcon,
   IntlConsumer,
 } from '@folio/stripes-core';
+import { PersistedPaneset } from '@folio/stripes-smart-components';
```

Comment:
This needs to be `from '../../PersistedPaneset'`; the current package already is `@folio/stripes-smart-components`.